core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -93,6 +93,15 @@ void modifyProperties(Consumer<Map<String,String>> mapMutator) throws AccumuloEx
    */
   Map<String,String> getSystemConfiguration() throws AccumuloException, AccumuloSecurityException;
 
+  /**
+   * Retrieves the configured System properties from zookeeper. This method is different from
+   * {@link #getSystemConfiguration()} as it will only return the configured properties in zookeeper
+   * and not properties from accumulo.properties or defaults.
+   *
+   * @return a map of system properties set in zookeeper
+   */
+  Map<String,String> getSystemProperties() throws AccumuloException, AccumuloSecurityException;
```

Comment:
Would renaming this to something like `getStoredProperties` help in highlighting the difference between the methods?

Reply:
Sure, we can rename it to something else if it makes it easier to distinguish the the difference as there was confusion around it. I think it's important to expose the method because with the new API you can mutate multiple properties at once and it's going to be very useful to query what properties were set vs what are defaults so the user can know what's been changed, etc.

Reply:
I'd rather keep `getSystemProperties` vs using `getStoredProperties`. If someone wants to know the difference, then they can read the javadoc. The word `stored` doesn't really convey the difference, the user still has to read the javadoc.

Reply:
> query what properties were set vs what are defaults so the user can know what's been changed, etc.

I'm confused. If the new `modifyProperties()` does not throw an exception, then won't all the properties passed in be set? From a user perspective, is there a situation where the properties I pass to the method don't all get set?